python/semantic_kernel/connectors/ai/open_ai/services/open_ai_chat_completion.py

```diff
@@ -55,14 +56,14 @@ def __init__(
                 env_file_path=env_file_path,
                 env_file_encoding=env_file_encoding,
             )
-        except ValidationError as ex:
+        # currently really difficult to trigger since all are optional
```

Comment:
Maybe by providing an incorrect type to some fields?

Reply:
good call, found that if passing a dict to it, it does throw validationerror, fixed now!